installer/scripts/npm_startup.sh

```diff
@@ -72,3 +74,7 @@ else
         echo "File $release_file not found!"
 fi
 
+if [ $require_notify -eq 1 ];
+then
+    echo "Please configure your firewall (if running), to allow connections for destination port 8084 over TCP for NPM solution to run"
```

Comment:
Yes it shall be always 8084. We have not yet had support for altering the port in the solution automatically. That sort of a request will require manual efforts which are not yet supported fully.